docx/oxml/text/run.py

```diff
@@ -98,6 +99,10 @@ def text(self):
                 text += '\t'
             elif child.tag in (qn('w:br'), qn('w:cr')):
                 text += '\n'
+            elif child.tag == qn('w:instrText'):
+                if child.text.lower().startswith(('tc \\l1 "', 'tc \\l2 "', 'tc \\l3 "', 'tc \\l4 "')):
+                    #paragraph form this run continues with `hidden text` for Table content
```

Comment:
Are these `tc \\l1`, `tc \\l2`, `tc \\l3` and `tc \\l4` only values that can occur? For example, can it be `tc \\lN`?

Reply:
Yes only these values, they represent the heading level.